### PR TITLE
Add perf tests for Azure.Search.Documents

### LINKED ISSUE #19009: Add performance tests
Implement perf tests for the Search scenarios

### PR DESCRIPTION
**Pull request by Mohit-Chakraborty**

### CODE REVIEW

File: sdk/search/Azure.Search.Documents/perf/README.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+
+When you submit a pull request, a CLA-bot will automatically determine whether you need to provide a CLA and decorate the PR appropriately (e.g., label, comment). Simply follow the instructions provided by the bot. You will only need to do this once across all repos using our CLA.
+
+This project has adopted the [Microsoft Open Source Code of Conduct](https://opensource.microsoft.com/codeofconduct/). For more information see the [Code of Conduct FAQ](https://opensource.microsoft.com/codeofconduct/faq/) or contact [opencode@microsoft.com](mailto:opencode@microsoft.com) with any additional questions or comments.
+
+Please see our [contributing guide](https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/eventhub/Azure.Messaging.EventHubs/CONTRIBUTING.md) for more information.
+
+![Impressions](https://azure-sdk-impressions.azurewebsites.net/api/impressions/azure-sdk-for-net%2Fsdk%2Feventhub%2FAzure.Storage.Files.DataLake.Perf%2FREADME.png)
```
Comment (jsquire) [inline sdk/search/Azure.Search.Documents/perf/README.md:15]: You've got a Data Lake path in there....

File: sdk/search/Azure.Search.Documents/perf/README.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+
+## Contributing
+
+This project welcomes contributions and suggestions.  Most contributions require you to agree to a Contributor License Agreement (CLA) declaring that you have the right to, and actually do, grant us the rights to use your contribution. For details, visit https://cla.microsoft.com.
+
+When you submit a pull request, a CLA-bot will automatically determine whether you need to provide a CLA and decorate the PR appropriately (e.g., label, comment). Simply follow the instructions provided by the bot. You will only need to do this once across all repos using our CLA.
+
+This project has adopted the [Microsoft Open Source Code of Conduct](https://opensource.microsoft.com/codeofconduct/). For more information see the [Code of Conduct FAQ](https://opensource.microsoft.com/codeofconduct/faq/) or contact [opencode@microsoft.com](mailto:opencode@microsoft.com) with any additional questions or comments.
+
+Please see our [contributing guide](https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/eventhub/Azure.Messaging.EventHubs/CONTRIBUTING.md) for more information.
```
Comment (jsquire) [inline sdk/search/Azure.Search.Documents/perf/README.md:13]: You probably want to link to the main contrib guide, not the Event Hubs one...

File: sdk/search/Azure.Search.Documents/perf/Microsoft.Azure.Search.Perf/Microsoft.Azure.Search.Perf.csproj [hunk 3]
```diff
@@ -0,0 +1,14 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
```
Comment (heaths) [inline sdk/search/Azure.Search.Documents/perf/Microsoft.Azure.Search.Perf/Microsoft.Azure.Search.Perf.csproj:1]: Why do anything with track 1 assemblies? (Same for file below)

**Review by heaths — APPROVED** [review]

File: sdk/search/Azure.Search.Documents/perf/Microsoft.Azure.Search.Perf/Microsoft.Azure.Search.Perf.csproj (hunk 3 above)
Comment (Mohit-Chakraborty) [reply sdk/search/Azure.Search.Documents/perf/Microsoft.Azure.Search.Perf/Microsoft.Azure.Search.Perf.csproj:1]: Thanks for catching this. This is the project file for the new perf tests for T1 APIs. They snuck in here while I have not finished writing the tests.
I am removing the T1 files from this PR.

File: sdk/search/Azure.Search.Documents/perf/README.md (hunk 1 above)
Comment (Mohit-Chakraborty) [reply sdk/search/Azure.Search.Documents/perf/README.md]: Fixed

**Review by jsquire — APPROVED** [review]

**Pull request merged**